docs/en/docs/lakehouse/multi-catalog/es.md

```diff
@@ -88,6 +88,50 @@ After switching to the ES Catalog, you will be in the `dafault_db`  so you don't
 | object        | string      |         |
 | other         | unsupported |         |
 
+### Array Type
+
+Elasticsearch does not have an explicit array type, but one of its fields can contain 
+[0 or more values](https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html).
+To indicate that a field is an array type, a specific `doris` structural annotation can be added to the 
+[_meta](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html) section of the index mapping.
```

Comment:
I recommend also add a lower version of the document reference
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/mapping-meta-field.html